### PR TITLE
test(button): add performance tests for mat-raised-button

### PR DESCRIPTION
**Pull request by wagnermaciel**
### Overview

* Test the render speed
* Test the click speed
* Small changes to checkbox benchmark

### CODE REVIEW

File: test/benchmarks/material/button/button.perf-spec.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    browser.rootEl = '#root';
+  });
+
+  it('renders a basic raised button', async() => {
+    await runBenchmark({
+      id: 'button-render',
+      url: '',
+      ignoreBrowserSynchronization: true,
+      params: [],
+      setup: async () => await $('#show').click(),
```
Comment (devversion) [inline test/benchmarks/material/button/button.perf-spec.ts:23]: Could we remove `setup` here, or do I miss something? Looks like we first click `show` and then for subsequent sampling iterations we call `#hide` before and then measure `#show`.

File: test/benchmarks/material/button/app.module.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+ * @title Button benchmark component.
+ */
+@Component({
+  selector: 'app-root',
+  template: `
+    <button id="show" (click)="show()">Show</button>
+    <button id="hide" (click)="hide()">Hide</button>
+
+    <ng-container *ngIf="isVisible">
+			<button mat-raised-button>Basic</button>
```
Comment (jelbourn) [inline test/benchmarks/material/button/app.module.ts:23]: You shouldn't need the `ng-container` ; you can put `ngIf` directly on the button

Comment (devversion) [reply test/benchmarks/material/button/app.module.ts:23]: yeah, or alternatively can just use `<ng-template [ngIf]="isVisible>` (if the grouping is intended)

I see that we are doing this in other approved/merged benchmarks too. We should most likely clean this up there too.

File: test/benchmarks/material/button/app.module.ts [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+  show() { this.isVisible = true; }
+  hide() { this.isVisible = false; }
+}
+
+
+@NgModule({
+  declarations: [ButtonBenchmarkApp],
+  imports: [
+    A11yModule,
```
Comment (devversion) [inline test/benchmarks/material/button/app.module.ts:41]: Consider removing `A11yModule`. I know I'm repeating this from other PRs, but it's just for tracking

File: test/benchmarks/material/button/app.module.ts [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+ * found in the LICENSE file at https://angular.io/license
+ */
+import {A11yModule} from '@angular/cdk/a11y';
+import {Component, NgModule, ViewEncapsulation} from '@angular/core';
+import {BrowserModule} from '@angular/platform-browser';
+import {MatButtonModule} from '@angular/material/button';
+
+/**
+ * @title Button benchmark component.
+ */
```
Comment (devversion) [inline test/benchmarks/material/button/app.module.ts:15]: While addressing other feedback, you could remove the `@title` comment here (as it's a leftover from doc examples)

**Comment by jelbourn** [comment]
You can add merge-ready once CI is green

**Pull request merged**